src/metabase/models/params.clj

```diff
@@ -59,10 +61,9 @@
   [param-field-ids]
   (not-empty
    (into {}
-         (map (comp (juxt :field_id identity)
-                    #(select-keys % [:field_id :human_readable_values :values])
-                    #(field-values/get-latest-full-field-values (:id %))))
-         (t2/hydrate (t2/select :model/Field :id [:in (set param-field-ids)]) :values))))
+         (comp (map (juxt identity field-values/get-latest-full-field-values))
+               (remove (comp nil? second)))
+         param-field-ids)))
```

Comment:
This is relatively unrelated. I've found out that incorrect `:param_values` were returned while writing the test and traced it to this function.